cg/meta/upload/scoutapi.py

```diff
@@ -49,16 +48,19 @@ def build_samples(self, analysis_obj: models.Analysis, hk_version_id: int = None
             sample_id = link_obj.sample.internal_id
             bam_path = self.fetch_file_path("bam", sample_id, hk_version_id)
             alignment_file_path = self.fetch_file_path("cram", sample_id, hk_version_id)
-            chromograph_path = self.fetch_file_path("chromograph", sample_id, hk_version_id)
+            chromograph_path = self.fetch_file_path(
+                "chromograph", sample_id, hk_version_id
+            )
             mt_bam_path = self.fetch_file_path("bam-mt", sample_id, hk_version_id)
-            vcf2cytosure_path = self.fetch_file_path("vcf2cytosure", sample_id, hk_version_id)
+            vcf2cytosure_path = self.fetch_file_path(
+                "vcf2cytosure", sample_id, hk_version_id
+            )
 
             lims_sample = dict()
```

Comment:
```suggestion
```

Reply:
This will break the code if we get an error in the try catch